DESIGN.md

```diff
@@ -60,7 +60,6 @@ and then converting to SCIP for transmission.
     and zstd are already very good in terms of both compression
     speed and compression ratio.[^3]
 - Support efficient code navigation by itself.
-
```

Comment:
This is an unrelated change.

Reply:
That's weird. Can you briefly look into why the check passed earlier? The CI has not been failing on `main`.

Reply:
Opened https://github.com/sourcegraph/scip/pull/328